Gemfile

```diff
@@ -30,7 +30,7 @@ group :development, :test do
   end
   gem 'capybara', '~> 2.4'
   gem 'database_cleaner', '~> 1.3'
-  gem 'factory_girl_rails', '~> 4.5'
+  gem 'factory_girl_rails', '~> 4.8'
```

Comment:
It installed 4.8 for me (which lead to the fail fixed in same commit). Still own commit?